test/simple-memory-corruption/uninitialized_read_large.c

```diff
@@ -0,0 +1,14 @@
+#include <stdlib.h>
+
+#include "../test_util.h"
+
+OPTNONE int main(void) {
+	char *p = malloc(128 * 1024);
```

Comment:
Still needs to be changed to 4 space tabs.

Reply:
Arg, my bad!